deps/ngtcp2/ngtcp2.gyp

```diff
@@ -79,19 +80,14 @@
   'lib/ngtcp2_map.h',
```

Comment:
Nit: The indentation is a little off here. I'd like to close https://github.com/nodejs/node/pull/33938 if we make the change here instead?

Reply:
Yeah I'll fix that up before landing. Right now I'm going through some of the persistent test failures to see if I can clean those up here also.